README.md

```diff
@@ -716,12 +724,13 @@ Before setting up the following ELEVATE-Project application, dependencies given
     -   **Ubuntu/Linux**
 
         ```
-        pm2 start ./project-service/app.js --name project-service && \
-        pm2 start ./entity-management/src/app.js --name entity-management && \
-        pm2 start ./user/src/app.js --name user && \
-        pm2 start ./notification/src/app.js --name notification && \
-        pm2 start ./interface-service/src/app.js --name interface && \
-        pm2 start ./scheduler/src/app.js --name scheduler
+        (cd project-service && pm2 start app.js --name project-service && cd -) && \
```

Comment:
@MallanagoudaB only this expected for native setup part in your documentation

Reply:
yes @VISHNUDAS-tunerlabs  only this changes was made by me .